src/app/pages/system/alert-service/alert-service-list/alert-service-list.component.html

```diff
@@ -0,0 +1,41 @@
+<mat-card>
+  <mat-toolbar-row>
+    <h3 class="card-title">{{ 'Alert Services' | translate }}</h3>
+
+    <div>
+      <ix-search-input [value]="filterString" (search)="filterUpdated($event)"></ix-search-input>
+      <ix-table-columns-selector [columns]="columns" (columnsChange)="columnsChange($event)"></ix-table-columns-selector>
+      <button
+        *ixRequiresRoles="requiresRoles"
```

Comment:
We don't add roles checks on buttons here, but instead add them in the form. We need to allow users to open the edit form because we may not be showing all the fields in the table. And then to keep UI consistent between edit and add scenarios, we should not have checks on this button.